src/sensors/TallyCounterI2C.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ *TallyCounterI2C.h
+ *This file is part of the EnviroDIY modular sensors library for Arduino
+ *
+ *Initial library developement done by Sara Damiano (sdamiano@stroudcenter.org).
+ *
+ *This file is for an external Event counter, used to measure windspeed or rainfall
+ *
+ *Documentation for the sensor can be found at:
+ * https://github.com/NorthernWidget-Skunkworks/Project-Tally​
+ * https://github.com/NorthernWidget-Skunkworks/Tally_Library/tree/Dev_I2C
+ *
+ * For Wind Speed:
+ *  Accuracy and resolution are dependent on the sensor used:
+ *  Inspeed WS2R Version II Reed Switch Anemometer
+ *    https://www.store.inspeed.com/Inspeed-Version-II-Reed-Switch-Anemometer-Sensor-Only-WS2R.htm
+ *
+ * Assume sensor is immediately stable
+*/
+
+#include "TallyCounterI2C.h"
+
+
+// The constructor - because this is I2C, only need the power pin and rain per event if a non-standard value is used
+TallyCounterI2C::TallyCounterI2C(uint8_t i2cAddressHex)
+     : Sensor("TallyCounterI2C", TALLY_NUM_VARIABLES,
+              TALLY_WARM_UP_TIME_MS, TALLY_STABILIZATION_TIME_MS, TALLY_MEASUREMENT_TIME_MS,
+              -1, -1, 1)
```

Comment:
Can the sensor ever be powered down?

Reply:
Generally counting devices such as this and the [TippingBucketRainCounter](https://github.com/EnviroDIY/TippingBucketRainCounter) need to always be "on", so that they can count an event. This is why I set `powerPin = -1` and `measurementsToAverage = 1`. 

However, @bschulz1701 designed the Tally device with a super capacitor, so once that is charged it can be powered down between readings.

I'll add the powerPin as an option.

Reply:
Fixed with commits 3f8c45df and f0823303